spec/plugin_mixins/rabbitmq_connection_spec.rb

```diff
@@ -26,6 +26,18 @@ def register
   }
   let(:hare_info) { instance.instance_variable_get(:@hare_info) }
 
+  describe "rabbitmq_settings" do
+    let(:rabbitmq_settings) { super.merge({"connection_timeout" => 123, "heartbeat" => 456}) }
```

Comment:
This merges it with the higher-up definition of `rabbitmq_settings`. I think @ph taught me that, or was it @purbon?